.env

```
@@ -42,7 +42,13 @@ NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
 
 
 NEXTAUTHURL = "http://localhost:3000"
+<<<<<<< HEAD
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The <<<<<<< HEAD key is duplicated

(DuplicatedKey)